src/Pos/Launcher/Runner.hs

```diff
@@ -199,6 +203,7 @@ runRealModeDo discoveryCtx slottingCtx transport np@NodeParams {..} sscnp
                runProduction .
                    runResourceT .
                    usingLoggerName lpRunnerTag .
+                   runJsonLogT' mJLHandle .
```

Comment:
@brunjlar I see that `JsonLogT` stores `Maybe (MVar Handle, JLTimedEvent -> IO Bool)` internally. So, as far as I understand, it means that here both monad stacks will use same handle but different `MVar`, so it may happen that they will print to same handle concurrently. I find it suspicious.
If `ekg` doesn't need JSON logging, then why not use `runWithoutJsonLogT`? If it needs JSON logging, then how do we solve the problem I described?